doc/source/whatsnew/v0.20.0.txt

```diff
@@ -1,45 +1,48 @@
-.. _whatsnew_0200:
+.. _whatsnew_0201:
 
-v0.20.0 (May 4, 2017)
+v0.20.1 (May 5, 2017)
 ---------------------
 
+This is a combined release for 0.20.0 and and 0.20.1.
+Version 0.20.1 contains one additional change for backwards-compatibility with downstream projects using pandas' hashing routines. (:issue:`16250`)
```

Comment:
to be correct it are the utils routines, not hashing (the hashing you asked before to add a deprecation shim, which we did)